conf.d/health.d/beanstalkd.conf

```diff
@@ -0,0 +1,33 @@
+# get the number of buried jobs in all queues
+
+template: buried_jobs
+      on: beanstalk.current_jobs
+    calc: $buried
+   units: jobs
+   every: 10s
+    warn: $this > 0
+    crit: $this > 10
+    info: the number of buried jobs aggregated across all tubes
+      to: sysadmin
```

Comment:
Please add `delay` line. The way it is now, it will spam you with alarms if this flips-flops.